### PR TITLE
Fix #176633 trancehost.com

### LINKED ISSUE #176633: www.trancehost.com
### Issue URL (Anti Adblock Script)
 [https://www.trancehost.com/blog/category/tutorials](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.trancehost.com%2Fblog%2Fcategory%2Ftutorials)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by stanislav-atr**
`trancehost.com#%#//scriptlet('prevent-xhr', 'static.doubleclick.net/instream/ad_status.js')` also works

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -3412,6 +3412,8 @@ topsporter.net#%#//scriptlet('abort-current-inline-script', '$', 'keepChecking')
 topsporter.net#%#//scriptlet('prevent-xhr', 'static.doubleclick.net/instream/ad_status.js')
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
 ||static.doubleclick.net/instream/ad_status.js$xmlhttprequest,redirect=nooptext,domain=topsporter.net
+! https://github.com/AdguardTeam/AdguardFilters/issues/176633
+||static.doubleclick.net/instream/ad_status.js$xmlhttprequest,redirect=nooptext,domain=trancehost.com
```
Comment (Alex-302) [inline BaseFilter/sections/antiadblock.txt:3416]: Scriptlet is better solution, than redirect - it helps when used DNS filtering. Please add to existing rule.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (stanislav-atr) [reply BaseFilter/sections/antiadblock.txt:3416]: so we should use both, right?

Comment (stanislav-atr) [reply BaseFilter/sections/antiadblock.txt:3416]: @Alex-302

Comment (Alex-302) [reply BaseFilter/sections/antiadblock.txt:3416]: Only `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=` and sriptlet.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**